numpy/lib/function_base.py

```diff
@@ -4206,7 +4209,8 @@ def insert(arr, obj, values, axis=None):
         # 2013-09-24, 1.9
         warnings.warn(
             "in the future the special handling of scalars will be removed "
-            "from insert and raise an error", DeprecationWarning)
+            "from insert and raise an error", DeprecationWarning,
+			stacklevel = 2)
```

Comment:
I think this should be aligned with the beginning of the string, not with the opening parenthesis.